core/templates/dev/head/pages/header_js_libs.html

```diff
@@ -14,3 +14,24 @@
 <script src="/third_party/static/headroom-js-0.9.4/angular.headroom.min.js"></script>
 <script src="/third_party/static/angular-drag-and-drop-lists-2.1.0/angular-drag-and-drop-lists.min.js"></script>
 <script src="/third_party/static/angular-recorder-1.4.1/dist/angular-audio-recorder.min.js"></script>
+
+<script>
+  (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
+  (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
+  m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
+  })(window,document,'script','https://www.google-analytics.com/analytics.js','ga');
+
+  (function() {
+    fetch('/analytics').then(function(response) {
+      return response.text()
+    }).then(function (response) {
```

Comment:
I don!t think that you need second `.then` you can do directly `responseData = response.json()`.

Reply:
response.json() returns a Promise so if we use responseData = response.json(), the responseData will be a Response object which resolves with the data. That's why the second .then is there